packages.json

```diff
@@ -1,38 +1,38 @@
 [
```

Comment:
This file shouldn't be modified. These versions < Ballerina.toml versions is how the build script recognizes that it needs to release these

Reply:
ok. will revert.

Reply:
Reverted https://github.com/wso2/choreo-apim-mediation-policies/pull/19/commits/8ae68c3d97f0c5f204ead171fff9bd7879be4a33